Makefile

```diff
@@ -108,10 +108,17 @@ clean-osm:
 chart-readme:
 	go run github.com/norwoodj/helm-docs/cmd/helm-docs -c charts -t charts/osm/README.md.gotmpl
 
-.PHONY: chart-checks
-chart-checks: chart-readme
+.PHONY: chart-check-readme
+chart-check-readme: chart-readme
 	@git diff --exit-code charts/osm/README.md || { echo "----- Please commit the changes made by 'make chart-readme' -----"; exit 1; }
 
+.PHONY: chart-check-values
+chart-check-values:
```

Comment:
Nit: could we rename this to something like `lint-chart` since `helm lint` does more than just check the values against the schema?

Reply:
Renamed to `helm-lint`